baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -607,6 +607,26 @@ export class EchoClient {
     this.initialize();
     return this.innerApiCalls.wait(request, options, callback);
   }
+/**
+ * This method is for decoding operations.
+ * @param {String} name
+ *   The operation name that will be passed.
+ * @returns {Promise} - The promise which resolves to an array.
+ *   The first element of the array is an object representing the result of operations.
+ *   The second element of the array is an object representing the metadata of operations.
+ *
+ * @code sample:
+ *   const decodedOperation = await waitDecodeLRO(name);
+ *   console.log(decodedOperation.result);
+ *   console.log(decodedOperation.metadata);
+ *
+ */
+  async checkwaitProgress(name: string): Promise<LROperation<protos.google.showcase.v1beta1.WaitResponse, protos.google.showcase.v1beta1.WaitMetadata>>{
```

Comment:
We need to `PascalCase` the method name here (I think we have a helper to do it) so that it becomes `checkWaitProgress`.

Reply:
you are right we have `toPascalCase()`, thanks!